src/algorithms/tracking/ActsExamples/EventData/MeasurementCalibration.hpp

```diff
@@ -0,0 +1,72 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2023 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/EventData/MultiTrajectory.hpp"
+#include "Acts/EventData/SourceLink.hpp"
+#include "Acts/EventData/VectorMultiTrajectory.hpp"
+#include "ActsExamples/EventData/Cluster.hpp"
```

Comment:
```
In file included from src/algorithms/tracking/CKFTracking.cc:35:
src/algorithms/tracking/ActsExamples/EventData/MeasurementCalibration.hpp:14:10: fatal error: ActsExamples/EventData/Cluster.hpp: No such file or directory
   14 | #include "ActsExamples/EventData/Cluster.hpp"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

Reply:
Interesting... this compiles fine for me :-D

Reply:
This will require `ACTS_BUILD_EXAMPLES=ON`.